RELEASE-NOTES.md

```diff
@@ -65,7 +63,7 @@ Create a directory and unpack the downloaded package:
 ```
 mkdir /tmp/node
 cd /tmp/node
-tar xf ~/Downloads/epoch-0.5.0-osx-10.12.6.tar.gz
+tar xf ~/Downloads/epoch-0.6.0-osx-10.12.6.tar.gz
```

Comment:
@dincho Is this suffix 10.12.6 confirmed? (Not a big deal - just for not confusing users.)

Reply:
yes, you can take a look https://circleci.com/gh/aeternity/epoch/442#artifacts/containers/0
However, we'll upload the binaries manually (or by still running Travis, if it gets it's job on time)